### PR TITLE
Hacktoberfest

### PR DESCRIPTION
**Pull request by ArnavHP18**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

added polynomial regression in python ml

### Checklist

- [ y] I've read the contribution guidelines.
- [y ] I've checked the issue list before deciding what to submit.
- [y ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: Python/ml/Polynomial_Regression.ipynb [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Polynomial Regression\n",
+    "from sklearn.preprocessing import PolynomialFeatures\n",
+    "poly_reg = PolynomialFeatures(degree=5)\n",
+    "X_poly = poly_reg.fit_transform(X ,y)"
```
Comment (Abhijit2505) [inline Python/ml/Polynomial_Regression.ipynb:177]: You have taken the degree of the polynomial regression as 5, as this was a small dataset it was easy to determine the degree.

I would like you to implement the same from the `car price prediction` dataset on Kaggle and also come up with a method to get the degree.

Hint - In the polynomial regression equation, which is the most significant term?

File: Python/ml/Polynomial_Regression.ipynb [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (Abhijit2505) [inline Python/ml/Polynomial_Regression.ipynb:181]: One more thing to keep in mind while preparing any regression model,
don't forget to make an EDA.

**Review by Abhijit2505 — CHANGES_REQUESTED** [review]
Although the main algorithm is correct, there are few questions,

1 - The dataset is small, having only two columns, the real-time dataset doesn't come like this and to perform polynomial regression we need to tweak the data.
2 - Add a method to determine the degree of the polynomial equation.

**Comment by atarax665** [comment]
@ArnavHP18 We haven't heard from you you in a while. Do the suggested changes ASAP to get this PR merged.

**Comment by HarshCasper** [comment]
I'm closing the PR due to no activity. If you would like to reopen this PR to be reviewed please join the [Slack](https://join.slack.com/t/tesseractcodi-bxq5968/shared_invite/zt-ju6c0nqq-KgkTvFyxjNVbDRBIjADanw) and drop me a message.

**Pull request closed without merging**